assets/src/edit-story/components/dialog/index.js

```diff
@@ -106,6 +107,11 @@ Dialog.propTypes = {
     PropTypes.node,
   ]),
   children: PropTypes.node,
+  maxWidth: PropTypes.string,
+};
+
+Dialog.defaultProps = {
+  maxWidth: '920px',
```

Comment:
Generally throughout the code, only the number comes through the attribute, and the `px` is added in the `styled` -- maybe we could do the same here? (At least that has been my impression that we've generally been doing that, LMK if you think otherwise.) Thoughts?

Reply:
I was first doing the same, but then I thought perhaps someone wanted to do `maxWidth: 100%`. In hindsight, that seems quite unrealistic though...

Reply:
Fixed now